sealed-secrets/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: sealed-secrets
+title: "Sealed Secrets"
+version: v0.12.4
+maintainer: hello@bitnami.com
```

Comment:
The only concern is the maintainer. This is supposed to be the community member that is agreeing to keep up to date with new releases, potentially help users if it's not working. I assume hello@bitnami.com knows nothing about adding this to our marketplace?

Reply:
Yeah, they don't know nothing about your marketplace but they do provide support to the 0.12.4 release. I assumed that the maintainer field was about "who maintains the software" and not "who maintains the software in the catalog".

Reply:
Thanks for this, I've tweaked the wording on the project README to make this clearer. Are you happy changing this to your twitter/email address?

Reply:
Since this is a community-contributed marketplace entry, and not from them, I do suggest putting your name in it.

Reply:
Feel free to add my name if you prefer but that may be misleading as I won't provide support to the software.

Reply:
It's not supporting the software, just supporting the installation of it on to k3s clusters using the marketplace. We aren't after a five-nines SLA with response times, just a general commitment to "I'll keep an eye on Civo's community Slack for if someone's talking about it, or if I'm @-mentioned and try to help as best I can, as soon as I can".

Unfortunately, if a marketplace contributor can't commit to that, we unfortunately can't add the application to our marketplace (otherwise we'd have to support applications we have no experience of using). Please let me know your thoughts @soukron. Thanks

Reply:
Yeah, as I said, go ahead and add my name.